### DISCUSSION TITLE
how to disable save button in the form in case of edit

### DISCUSSION THREAD

**Discussion by sivaputrevu** [react-hook-form/react-hook-form#2819] [General]
Hi Bill,

I have a question on to disable save button in the form in case of edit where all the form data will get populated from it's parent.
errors object would also be undefined in this case[onLoad]. is there any way we can handle formState?

in case of create along with errors object I created a local variable which handles onLoad save disable. do I need to handle the same local variable using any object from react-hook-form?

thanks in advance!

**Comment by bluebill1049** [comment]
for loading external data from an api, you would have to handle that at your app level.

**Reply by sivaputrevu** [reply]
nope not from API the data is being passed from one react component to another component
